catalyst-gateway/tests/api_tests/integration/test_config_frontend.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from api.v1 import config_frontend
+
+
+def test_config_frontend():
```

Comment:
Not sure whether it is necessary to add ip test here. I assume there will be no specific config for ip address, so maybe add TODO

In case this test is needed. As I remember it should be automatically added to either `x-real-ip` `forwarded` or `x-forwarded-for`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/X-Forwarded-For